Remove dependencies on xwin and zig.  We're not offering cross-compilation
options using these crates.

diff --git a/Cargo.toml b/Cargo.toml
index 6cbdca3..22ea5ef 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -76,16 +76,6 @@ version = "0.1.4"
 [dependencies.cargo-options]
 version = "0.6.0"
 
-[dependencies.cargo-xwin]
-version = "0.14.3"
-optional = true
-default-features = false
-
-[dependencies.cargo-zigbuild]
-version = "0.16.10"
-optional = true
-default-features = false
-
 [dependencies.cargo_metadata]
 version = "0.15.3"
 
@@ -310,8 +300,6 @@ version = "4.3.0"
 [features]
 cli-completion = ["dep:clap_complete_command"]
 cross-compile = [
-    "zig",
-    "xwin",
 ]
 default = [
     "full",
@@ -330,7 +318,6 @@ log = ["tracing-subscriber"]
 native-tls = [
     "dep:native-tls",
     "ureq?/native-tls",
-    "cargo-xwin?/native-tls",
     "dep:rustls-pemfile",
 ]
 password-storage = [
@@ -340,7 +327,6 @@ password-storage = [
 rustls = [
     "dep:rustls",
     "ureq?/tls",
-    "cargo-xwin?/rustls-tls",
     "dep:rustls-pemfile",
 ]
 scaffolding = [
@@ -358,5 +344,3 @@ upload = [
     "wild",
     "dep:dirs",
 ]
-xwin = ["cargo-xwin"]
-zig = ["cargo-zigbuild"]
